tgbotapi.extensions.utils/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/utils/extensions/venue/Foursquare.kt

```diff
@@ -16,7 +16,7 @@ fun Venue(
     title: String,
     address: String,
     foursquare: Foursquare
```

Comment:
I am not pretty sure that it is impossible to use `google_place_id` and `google_place_type` with foursquare. So, I think we must add these parameters too

Reply:
But there is `Google.kt` for that